testing/test-scripts/k8s/default_routes.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 IBM Corporation
```

Comment:
we were working on these concurrently ;-(
May not be needed after a merge of #514 (which matches the bookinfo walkthrough files)